docs/source/usage/cluster.rst

```diff
@@ -0,0 +1,148 @@
+Installing a Cluster on ESI
+===========================
+
+A lessee wishing to create a cluster out of multiple leased nodes and a private network can do so by running the correct `CLI commands`_. Alternatively, they can use the cluster CLI commands in `python-esiclient`_ to simplify the workflow involved in creating and managing their clusters.
```

Comment:
I'm not quite clear about the relationship between `openstack esi cluster orchestrate` and openstack esi openshift orchestrate`

Reply:
Oh, they're completely independent.